sdk/servicebus/azure-messaging-servicebus/src/samples/README.md

```diff
@@ -30,18 +30,24 @@ connection string value can be obtained by:
 
 ## Examples
 
+### Asynchronously sending and receiving
+
 - [Send messages][SendMessageAsyncSample]
```

Comment:
Should we call it "Send One Message" ? Specially ,, User should not think that they can send many messages using this API.